src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -117,9 +146,23 @@ export default {
 	computed: {
 		messageTime() {
 			return OC.Util.formatDate(this.timestamp * 1000, 'LT')
+		},
+		quote() {
+			if (this.parent) {
+				return this.$store.getters.message(this.token, this.parent)
+			} else return undefined
```

Comment:
Don't mix the two syntaxes.
```suggestion
			}
			return undefined
```

Reply:
Or even better
```suggestion
			return this.token && this.$store.getters.message(this.token, this.parent)
```

Reply:
> Or even better

Isn't that less readable despite being more concise?

Reply:
Not to my opinion :)
You can put it in a new line.

```js
return this.parent
	&& this.$store.getters.message(this.token, this.parent)
```